docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst

```diff
@@ -25,3 +27,104 @@ for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
 
 .. todo::
    Describe cause model
+
+Data Descriptions
+-----------------
+
+Preliminary!
+
+.. list-table:: Definitions
+	:widths: 5 10 10
+	:header-rows: 1
+	
+	* - State
+	  - State name
+	  - Definition
+	* - C
+	  - With **C**\ ondition
+	  - Simulant was born with haemolytic disease or other neonatal jaundice
+	* - F
+	  - **F**\ ree of condition
+	  - Simulant was not born with haemolytic disease or other neonatal jaundice
+	  
+.. list-table:: States
+	:widths: 5 10 10 20
+	:header-rows: 1
+	
+	* - State
+	  - Measure
+	  - Value
+	  - Notes
+	* - C
+	  - prevalence
+	  - prev_c384
+	  -
+	* - C
+	  - birth prevalence
+	  - birth_prev_unknown
```

Comment:
I think this should just be birth_prevalence_c384, and then in the data sources table, we explain how to get it. Or, if we can't directly pull birth prevalence (e.g. if we need to pull count data and divide by births), put that calculation here, and explain how to pull the values we need in the data sources table.